First draft workshop content (#3)
observability-lab-cse/observability-lab-workshop: troubleshoot/README.md

# Troubleshoot

> ## KV deployement

Reviewer: ```suggestion
## KV deployment
```

Also, what is KV?

> Re-deployment fails due to deleted KV with same name.

Reviewer: ```suggestion
Re-deployment fails due to a deleted Key Vault with the same name.
```

> **Solution:** Purge or recover old KV

Reviewer: ```suggestion
**Solution:** Purge or recover the old Key Vault
```
